modules/telemetry-otel/build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Please move it to `plugins`, we should not install experimental features by default